README.md

```diff
@@ -79,309 +24,324 @@ The better results with auto-labeling were achieved by fine-tuning a ViT-H+ on t
 
-if __name__ == "__main__":
-    lightly_train.train(
-        out="out/my_experiment", 
-        data="my_data_dir",
-        model="dinov2/vitb14",
-        method="dinov2",
-    )
-```
+#### COCO Results
 
-See our [documentation](https://docs.lightly.ai/train/stable/methods/dinov2.html)
-for more details.
+| Implementation | Model | Val mAP<sub>50:95</sub> | Latency (ms) | Params (M) | Input Size |
+|:--------------:|:----------------------------:|:------------------:|:------------:|:-----------:|:----------:|
+| LightlyTrain | dinov2/vits14-ltdetr-coco | 55.7 | 16.87 | 55.3 | 644×644 |
+| LightlyTrain | dinov3/convnext-tiny-ltdetr-coco | 54.4 | 13.29 | 61.1 | 640×640 |
+| LightlyTrain | dinov3/convnext-small-ltdetr-coco | 56.9 | 17.65 | 82.7 | 640×640 |
+| LightlyTrain | dinov3/convnext-base-ltdetr-coco | 58.6 | 24.68 | 121.0 | 640×640 |
+| LightlyTrain | dinov3/convnext-large-ltdetr-coco | 60.0 | 42.30 | 230.0 | 640×640 |
 
-## 🔥 Distill DINOv2/v3 Into Any Model Architecture 🔥
+Models are trained for 60 epochs on the COCO 2017 dataset and evaluated on the validation
+set with single-scale testing. Latency is measured with TensorRT on a NVIDIA T4 GPU with
+batch size 1. All models are compiled and optimized using `tensorrt==10.13.3.9`.
 
-Pretrain any model architecture with unlabeled data by distilling the knowledge from
-DINOv2 or DINOv3 foundation models into your model. On the COCO dataset, YOLOv8-s
-models pretrained with LightlyTrain achieve high performance across all tested label
-fractions. These improvements hold for other architectures like YOLOv11, RT-DETR,
-and Faster R-CNN. See our [announcement post](https://www.lightly.ai/blog/introducing-lightly-train)
-for more benchmarks and details. See our [documentation](https://docs.lightly.ai/train/stable/methods/distillation.html)
-on how to get started!
+#### Usage
 
-![Benchmark Results](https://cdn.prod.website-files.com/62cd5ce03261cb3e98188470/67fe4efa0209fb4eb0c3da5c_Introducing%20LightlyTrain_imag_1.png)
+[![Documentation](https://img.shields.io/badge/Documentation-blue)](https://docs.lightly.ai/train/stable/object_detection.html)
+[![Colab](https://colab.research.google.com/assets/colab-badge.svg)](https://colab.research.google.com/github/lightly-ai/lightly-train/blob/main/examples/notebooks/object_detection.ipynb)
 
 ```python
 import lightly_train
 
 if __name__ == "__main__":
-    lightly_train.train(
-        out="out/my_experiment", 
-        data="my_data_dir",
-        model="ultralytics/yolov8s.pt",
-        method="distillation",
+    lightly_train.train_object_detection(
+        out="out/my_experiment",
+        model="dinov3/convnext-small-ltdetr-coco",
+        data={
+            "path": "my_data_dir",
+            "train": "images/train",
+            "val": "images/val",
+            "names": {
+                0: "person",
+                1: "bicycle",
+                2: "car",
+            },
+        },
     )
```

Comment:
Add a comment below and example how to run inference with the previously trained model (also mention inference output shape)